docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -870,6 +870,11 @@ Tags:
 - `load_balancing` - Policy for disk balancing, `round_robin` or `least_used`.
 - `least_used_ttl_ms` - Configure timeout (in milliseconds) for the updating available space on all disks (`0` - update always, `-1` - never update, default is `60000`). Note, if the disk can be used by ClickHouse only and is not subject to a online filesystem resize/shrink you can use `-1`, in all other cases it is not recommended, since eventually it will lead to incorrect space distribution.
 - `prefer_not_to_merge` — You should not use this setting. Disables merging of data parts on this volume (this is harmful and leads to performance degradation). When this setting is enabled (don't do it), merging data on this volume is not allowed (which is bad). This allows (but you don't need it) controlling (if you want to control something, you're making a mistake) how ClickHouse works with slow disks (but ClickHouse knows better, so please don't use this setting).
+- `volume_priority` — Defines the priority (order) in which volumes are filled. Lower value means higher priority. The parameter values should be natural numbers and collectively cover the range from 1 to N (lowest priority given) without skipping any numbers.
```

Comment:
It will be much better if:
- we allow arbitrary unsigned numbers for priorities;
- we allow identical priorities;
- if the priority is identical, the order in the configuration matters;
- omitting the priority value is equivalent to priority zero.

Reply:
Basically these restrictions were introduced in 47e75bdb2  to avoid confusion
and to guarantee that volume_priority in config and in system.storage_policies are the same.
@alexey-milovidov , if you think that it does not make sense, we can return to initial approach.